src/cc/tests/integtest.cc

```diff
@@ -83,23 +93,33 @@ QFSTest::TearDown()
 
 }
 
-string
-QFSTestUtils::WriteTempFile(const string& data)
+int
+QFSTestUtils::CreateTempFile(string* path)
 {
      string filename = QFSTestUtils::kTestHome + "/tmp_file.XXXXXX";
      int fd = mkstemp(const_cast<char*>(filename.c_str()));
      EXPECT_NE(-1, fd) << "Could not mkstep: " << strerror(errno);
```

Comment:
Is this a common usage? "EXPECT_NE(...) << ... << ..."
Ignore the comment if it is.